### PR TITLE
Merge all sections within the raw data section into one and provide examples

### PR DESCRIPTION
**Pull request by Steffengreiner**

### CODE REVIEW

File: docs/rawdata/raw_data_download.md [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+2. Navigate to the raw data summary [view](#raw-data-navigation)
+3. Acquire [download URLs](#raw-data-url-generation) for the registered measurement raw data. 
+4. Start the measurement raw data [download](#download-raw-data) measurement and get a coffee :coffee:
+
+## Personal access token
+
+Before you can begin with downloading any data via HTTPS, you need to tell the download server
+who you are. We enforce a token-based authentication (personal access token: PAT) so you are not
+required to expose you password
+with anyone or with any system.
```
Comment (wow-such-code) [inline docs/rawdata/raw_data_download.md:19]:
```suggestion
required to expose your password
to anyone or to any system.
```

File: docs/rawdata/raw_data_download.md [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+![generate token](images/raw_data_create_pat_generate_token.png)
+
+!!! warning "Token accessibility"
+    A generated token will be only visible in its raw notation once! Make sure to store it safely in
+    your local password manager, you will not be able to access the token value again.
+
+### Manage tokens
+
+You can see your created tokens in the overview, however since they are instantly encrypted after generation,
+you cannot access their raw value anymore. We encourage you to use meaningful descriptions for your tokens,
```
Comment (wow-such-code) [inline docs/rawdata/raw_data_download.md:46]: do people understand "raw value" in this context?

File: docs/rawdata/raw_data_download.md [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+so you remember for what purpose you have created them.
+
+![token overview](images/raw_data_create_pat_token_overview.png)
+
+!!! danger "Token security"
+    If you are unsure, if your PAT got exposed or shared with untrusted parties, delete them right
+    away.
```
Comment (wow-such-code) [inline docs/rawdata/raw_data_download.md:53]:
```suggestion
    away. You can create new, safe tokens at any time.
```

File: docs/rawdata/raw_data_download.md [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+![raw_data_summary_no_data.png](images/raw_data_summary_no_data.png)
+
+### Raw Data URL Generation
+
+To acquire the URLS necessary for the measurement data download the following steps have to be taken:
+
+1. Navigate to the [raw data summary view](#raw-data-navigation)
+2. Select the measurements for which the raw data URLS should be generated.
+   ![raw_data_download_measurement_selection.png](images/raw_data_download_URL_generation_measurement_selection.png)
+3. Press the "Generate URL List" button to download a text file containing the URLs of the selected measurements
```
Comment (wow-such-code) [inline docs/rawdata/raw_data_download.md:73]: generate or download?

File: docs/rawdata/raw_data_download.md [hunk 5]
```diff
@@ -0,0 +1,178 @@
...
+
+This will take you to the raw data summary view
+![raw_data_summary_no_data.png](images/raw_data_summary_no_data.png)
+
+### Raw Data URL Generation
+
+To acquire the URLS necessary for the measurement data download the following steps have to be taken:
+
+1. Navigate to the [raw data summary view](#raw-data-navigation)
+2. Select the measurements for which the raw data URLS should be generated.
```
Comment (wow-such-code) [inline docs/rawdata/raw_data_download.md:71]:
```suggestion
2. Select the measurements for which the raw data URLs should be generated.
```

File: docs/rawdata/raw_data_download.md [hunk 6]
```diff
@@ -0,0 +1,178 @@
...
+   ![raw_data_download_measurement_selection.png](images/raw_data_download_URL_generation_measurement_selection.png)
+3. Press the "Generate URL List" button to download a text file containing the URLs of the selected measurements
+   ![raw_data_download_URL_downloaded.png](images/raw_data_download_URL_downloaded.png)
+
+### Download Raw Data
+
+To download the raw data for your measurements basic command line/terminal knowledge is required.
```
Comment (wow-such-code) [inline docs/rawdata/raw_data_download.md:78]:
```suggestion
To download the raw data for your measurements, basic command line/terminal knowledge is required. 
```

File: docs/rawdata/raw_data_download.md [hunk 7]
```diff
@@ -0,0 +1,178 @@
...
+=== "wget"
+
+    ``` bash
+    wget --content-disposition --trust-server-names --header "Authorization: Bearer <ACCESS_TOKEN>" -i <file-with-urls>
+    ```
+
+!!! warning
+    To ensure character validity in the text file, please format it in the *UTF-8* format
+
+The easiest way to generate such a text file is to navigate into the raw data view and select the measurements for which you want to download the raw data for and click the "download url list" button.
```
Comment (wow-such-code) [inline docs/rawdata/raw_data_download.md:158]: NIT: this is a repetition from the explanation above

Comment (wow-such-code) [reply docs/rawdata/raw_data_download.md:158]: you may want to shorten it

**Review by wow-such-code — CHANGES_REQUESTED** [review]
some small comments

File: docs/rawdata/raw_data_download.md (hunk 4 above)
Comment (Steffengreiner) [reply docs/rawdata/raw_data_download.md:73]: Good spot!

File: docs/rawdata/raw_data_download.md (hunk 2 above)
Comment (Steffengreiner) [reply docs/rawdata/raw_data_download.md:46]: Good point i replace it !

File: docs/rawdata/raw_data_download.md (hunk 7 above)
Comment (Steffengreiner) [reply docs/rawdata/raw_data_download.md:158]: Good point i linked it to the section instead of repeating it

**Review by wow-such-code — APPROVED** [review]

**Pull request merged**